lib/active_shipping/carriers/ups.rb

```diff
@@ -399,23 +400,7 @@ def build_shipment_request(origin, destination, packages, options={})
             if options[:prepay]
               xml.PaymentInformation do
                 xml.Prepaid do
-                  xml.BillShipper do
-                    xml.AccountNumber(options[:origin_account])
-                  end
-                end
-              end
-            elsif options[:bill_third_party]
-              xml.PaymentInformation do
-                xml.BillThirdParty do
-                  xml.BillThirdPartyShipper do
-                    xml.AccountNumber(options[:billing_account])
-                    xml.ThirdParty do
-                      xml.Address do
-                        xml.PostalCode(options[:billing_zip])
-                        xml.CountryCode(options[:billing_country])
-                      end
-                    end
-                  end
+                  build_billing_info_node(xml, options)
```

Comment:
This will work differently than before if someone is passing both prepay and bill third party options.